tests/queries/tests.py

```diff
@@ -3825,3 +3825,17 @@ def test_ticket_23622(self):
             Ticket23605A.objects.filter(qx),
             [a2], lambda x: x
         )
+
+class Ticket26983Tests(TestCase):
```

Comment:
Could you try to find a better place for the test? The previous test uses that lame naming because it's truly some strange queries. In simpler cases, we usually try to use a more meaningful test name and it doesn't need its own test case.